### PR TITLE
InstructorSidebar.js--Feature/instructor logout button+function

### PR DESCRIPTION
**Pull request by pjaepole**
link to loom video explanation https://www.loom.com/share/9c781ac65b7247f09025dee4b2889769
highly recommend 2x playback speed 

- imported ExportOutlined from @ant-design/icons, imported { Link } from react-router-dom

>importing Header.js in index.js file to see if the prebuilt logout function works or not
after testing it was working so I will move over the function to the logout button on instructor and parent logout
removed index.js changes to revert back to original

- InstructorSidebar.js 
import { useOktaAuth } from '@okta/okta-react';
const { authService } = useOktaAuth();

- added onclick function to the logout menuItem to log out when clicked
does remove token from localstorage and it redirect user to login page!

- removed a modified file from a pull request, parentsidebar.js which does not have anything to do with this pull request

### CODE REVIEW

**Review by alextcedwards — CHANGES_REQUESTED** [review]
Good job separating the pull requests! couple things: 

Think you forgot to include video recording to explain what you did. 

Updating craco to different version will cause merge conflicts with everyone since no one else updated, so I would change that back to original.

The rest looks pretty good though!

**Comment by Ryan-E-Mark** [comment]
Looks great, good work referencing the Okta documentation to successfully implement the removal of the JWT from localstorage.

File: src/components/pages/InstructorHome/InstructorSidebar.js [hunk 1]
```diff
@@ -50,6 +53,13 @@ function InstructorSidebar() {
         <Menu.Item key="4" icon={<PlusOutlined />}>
           <a href="/instructor-add-course">Add Courses</a>
         </Menu.Item>
+
+        <Menu.Item 
+          key="logout" 
+          onClick={() => {authService.logout();}} 
+          icon={<ExportOutlined fontSize="150px" />}>
+            <Link to="/logout">Logout</Link>
```
[the remark this answers is not in the document]
Comment (pjaepole) [reply src/components/pages/InstructorHome/InstructorSidebar.js]: it seems like okta authService.logout automatically redirects the user to the login page after logging out.
But I agree that it is a better idea to change it to landing page.
I changed it to <Link to='/landing'> and I think I missed this on other PR as well so I will change that as well

[the remark this answers is not in the document]
Comment (Ryan-E-Mark) [reply src/components/pages/InstructorHome/InstructorSidebar.js]: Ahh ok. Well either way good job on this pull request! We can work through the okta stuff a bit more once Ryan gets back.

**Pull request merged**